server/models/index.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable import/order */
```

Comment:
Instead of creating a custom model, We can establish the database connection in the `server/index.js` file. This provides us with the possibility of shutting down the app if the database failed to connect.